### PR TITLE
Add config option for radiation

### PR DESCRIPTION
**Pull request by iTwins**
## Description

There was some discussion about people wanting to disable radiation.

## Proposed changes

Add a config option that controls whether the radiation task will be scheduled.

## Related Issues (if applicable)

N/A

## Checklist

- [x] I have fully tested the proposed changes and promise that they will not break everything into chaos.
- [ ] I have also tested the proposed changes in combination with various popular addons and can confirm my changes do not break them.
- [x] I have made sure that the proposed changes do not break compatibility across the supported Minecraft versions (1.16.* - 1.20.*).
- [x] I followed the existing code standards and didn't mess up the formatting.
- [x] I did my best to add documentation to any public classes or methods I added.
- [x] I have added `Nonnull` and `Nullable` annotations to my methods to indicate their behaviour for null values
- [ ] I added sufficient Unit Tests to cover my code.

### CODE REVIEW

File: src/main/java/io/github/thebusybiscuit/slimefun4/implementation/Slimefun.java [hunk 1]
```diff
@@ -367,7 +367,9 @@ private void onPluginStart() {
         // Armor Update Task
         if (config.getBoolean("options.enable-armor-effects")) {
             new SlimefunArmorTask().schedule(this, config.getInt("options.armor-update-interval") * 20L);
-            new RadiationTask().schedule(this, config.getInt("options.radiation-update-interval") * 20L);
+            if (config.getBoolean("options.enable-radiation")) {
+                new RadiationTask().schedule(this, config.getInt("options.radiation-update-interval") * 20L);
+            }
```
Comment (TheBusyBiscuit) [inline src/main/java/io/github/thebusybiscuit/slimefun4/implementation/Slimefun.java:372]: Should this be moved outside of the enable-armor-effects condition?

Comment (iTwins) [reply src/main/java/io/github/thebusybiscuit/slimefun4/implementation/Slimefun.java:372]: The reason this was inside the enable-armor-effects condition is probably that the hazmat suit wouldn't work if armor effects were disabled, making it impossible to handle radioactive items if the radiation task wouldn't also be disabled.

**Review by J3fftw1 — DISMISSED** [review]

File: src/main/java/io/github/thebusybiscuit/slimefun4/implementation/Slimefun.java (hunk 1 above)
Comment (TheBusyBiscuit) [reply src/main/java/io/github/thebusybiscuit/slimefun4/implementation/Slimefun.java:372]: Therefore `options.enable-armor-effects` = false and `options.enable-radiation` = true is an invalid state.
I would strongly suggest to add a console warning in this case and warn that radiation will not work unless armor effects are enabled.

Comment (iTwins) [reply src/main/java/io/github/thebusybiscuit/slimefun4/implementation/Slimefun.java:372]: Added a warning for the mentioned state

**Review by JustAHuman-xD — APPROVED** [review]
lgtm

**Review by J3fftw1 — APPROVED** [review]

**Review by WalshyDev — APPROVED** [review]

**Pull request merged**